packages/core/src/components/spinner/spinner.scss

```diff
@@ -0,0 +1,54 @@
+:host {
+  --p: 0;
```

Comment:
What does this variable do?

Also, just an idea (would probably be best as a separate PR), we could use [CSS parts](https://developer.mozilla.org/en-US/docs/Web/CSS/::part) to customize the spinner.